i18n/en.json

```diff
@@ -1642,8 +1642,10 @@
   "announcement_bar.error.site_url_gitlab.full": "Please configure your [Site URL](https://docs.mattermost.com/administration/config-settings.html#site-url) in the [System Console](/admin_console/environment/web_server) or in gitlab.rb if you're using GitLab Mattermost.",
   "announcement_bar.error.site_url.full": "Please configure your [Site URL](https://docs.mattermost.com/administration/config-settings.html#site-url) in the [System Console](/admin_console/environment/web_server).",
   "announcement_bar.notification.email_verified": "Email verified",
+  "api.channel.add_guest.added": "{addedUsername} added to the channel as guest by {username}.",
```

Comment:
Please update to: "{addedUsername} added to the channel as a guest by {username}."